types/node/fs/promises.d.ts

```diff
@@ -87,13 +87,6 @@ declare module "fs/promises" {
         highWaterMark?: number | undefined;
         flush?: boolean | undefined;
     }
-    interface ReadableWebStreamOptions {
-        /**
-         * Whether to open a normal or a `'bytes'` stream.
-         * @since v20.0.0
-         */
-        type?: "bytes" | undefined;
-    }
```

Comment:
This is removing an exported symbol, but its only conceivable use is in constructing an `options` parameter to `readableWebStream()` that itself no longer exists, so anyone using it is going to see some sort of error either way.